test/render.tsx

```diff
@@ -2,17 +2,17 @@ import {serial as test} from 'ava';
 import {spawn} from 'node-pty';
 import ansiEscapes from 'ansi-escapes';
 
-const term = (fixture, args = []) => {
-	let resolve;
-	let reject;
+const term = (fixture: string, args: string[] = []) => {
+	let resolve: (value?: any) => void;
+	let reject: (error: Error) => void;
 
 	// eslint-disable-next-line promise/param-names
 	const exitPromise = new Promise((resolve2, reject2) => {
 		resolve = resolve2;
 		reject = reject2;
 	});
 
-	const ps = spawn('node', ['./fixtures/run', `./${fixture}`, ...args], {
+	const ps = spawn('ts-node', [`./fixtures/${fixture}.tsx`, ...args], {
```

Comment:
```suggestion
	const ps = spawn(process.execPath, [`./fixtures/${fixture}.tsx`, ...args], {
```

?

Reply:
This doesn't work unfortunately because `process.execPath` is path to `node` not `ts-node`. Maybe because the process is being ran by ava?

Reply:
Ah, my fault. I missed that you were using `ts-node/register/transpile-only` and not `ts-node/register`.